service.datadog.yaml

```diff
@@ -0,0 +1,15 @@
+---
+schema-version: v2
+dd-service: datadog-agent-nightly
+
+extensions:
+  datadoghq.com/sdp:
+    workday_team: "Agent Platform"
+    conductor:
+      slack: "datadog-agent-pipelines"
+      schedule: "0 3 3 * * 1-5"
+      parameterized_deployment:
+        workflow_generator: ""
+      env:
+        - name: "staging"
+          branch: "chouquette/conductor"
```

Comment:
Just a double check, is it supposed to be this branch?

Reply:
Yes, in the sense that I expect some followup work, but I'd prefer to be able to iterate on a dedicated branch rather than on main.
Although I must confess I'm not sure how this will behave if the branch gets removed